releases/release-1.23/release-notes/maps/pr-104691-map.yaml

```diff
@@ -0,0 +1,8 @@
+pr: 104691
+releasenote:
+  text: |-
+    IPv6DualStack feature moved to stable.
+    Controller Manager flags for the node IPAM controller have slightly changed:
+    1. When configuring a dual-stack cluster, the user must specify both --node-cidr-mask-size-ipv4 and --node-cidr-mask-size-ipv6 to set the per-node IP mask sizes, instead of the previous --node-cidr-mask-size flag.
```

Comment:
```suggestion
    1. When configuring a dual-stack cluster, the user must specify both `--node-cidr-mask-size-ipv4` and `--node-cidr-mask-size-ipv6` to set the per-node IP mask sizes, instead of the previous `--node-cidr-mask-size` flag.
```